trustymail/cli.py

```diff
@@ -26,6 +26,7 @@
 Notes:
    If no scan type options are specified, all are run against a given domain/input.
 """
+from . import __version__
```

Comment:
I would tend to prefer the more explicit: `from trustymail import __version__`

Reply:
Agreed